src/components/snapshots/useSnapshot.ts

```diff
@@ -281,4 +283,15 @@ const copyFiles = async (snapshotFolder: string, fileRecords: ObjectRecord[] = [
   }
 }
 
+// Mutate the active config object to inject new preferences
+type Config = typeof config
+export const refreshPreferences = (config: Config) => {
+  const serverPrefs: ServerPreferences = JSON.parse(readFileSync(PREFERENCES_FILE, 'utf-8')).server
+  serverPrefKeys.forEach((key) => {
```

Comment:
At first I just iterated over the imported "serverPrefs", but the problem is if a key is absent from the imported server prefs, then it would get left unchanged in "config", when it actually should be erased so that system defaults are used.

So had to make an array of all possible server pref keys and iterate over that to ensure that *all* possible prefs will be updated (or removed).